plugin/core/views.py

```diff
@@ -984,6 +985,10 @@ def is_location_href(href: str) -> bool:
     return href.startswith("location:")
 
 
+def starts_with_custom_uri_scheme(href: str) -> bool:
```

Comment:
With this function name, shouldn't it exclude at least `file`, `res` and `subl` schemes too?

Reply:
I inlined the function